lib/yardstick/rule_description/formatter.rb

```diff
@@ -0,0 +1,47 @@
+module Yardstick
+  class RuleDescription
+    # Rule description text formatter
+    class Formatter
+      include Adamantium, Concord.new(:description)
+
+      # Map of token types to decorators
+      TOKEN_DECORATORS = {
```

Comment:
Yeah it was triggering this issue once I added `Adamantium`: https://github.com/dkubb/ice_nine/issues/20